src/components/Main/Schedule/NewScheduleForm.tsx

```diff
@@ -0,0 +1,179 @@
+import React, { useState } from 'react';
+import { FormField } from '../../shared/FormField';
+
+import { TextField, Select } from '@material-ui/core';
+import MenuItem from '@material-ui/core/MenuItem';
+import FormControl from '@material-ui/core/FormControl';
+import InputLabel from '@material-ui/core/InputLabel';
+import { makeStyles, Theme, createStyles } from '@material-ui/core/styles';
+import FormHelperText from '@material-ui/core/FormHelperText';
+// import InfoIcon from '@material-ui/icons/Info';
+
+// import ReactTooltip from 'react-tooltip';
+import { Tooltip } from '../../shared/Tooltip';
+
+// TODO hook up teams with data from DB
+
+interface NewScheduleFormProps {
+  onSubmit: (
+    scheduleTitle: string,
+    startDate: string,
+    endDate: string,
+    view: string,
+    team: number,
+  ) => void;
+  onClose: () => void;
+}
+
+export const NewScheduleForm = ({ onSubmit, onClose }: NewScheduleFormProps) => {
+  const [scheduleTitle, setScheduleName] = useState('');
+  const [startDate, setStartDate] = useState(toDateString(new Date()));
+  const [endDate, setEndDate] = useState(toDateString(new Date()));
+  const view = 'weekly';
+  const [team, setTeam] = useState<number>(0);
+  const [errorMessage, setErrorMessage] = useState('');
+  const classes = useStyles();
+
+  //needed to format date so that the date picker can display it properly
+  function toDateString(date: Date): string {
+    return `${date.getFullYear()}-${date.getMonth()}-${date.getDate()}`;
+  }
+
+  function NewlineText(props: { text: string }) {
+    const text = props.text;
+    const newText = text.split('\n').map((str) => <p>{str}</p>);
```

Comment:
If `\n` isn't working maybe just do this?
```suggestion
    const newText = text.replaceAll('\n', '<br/>');
```

Reply:
I got a new way of handling validation and form input errors, but i'll try to remember this for next time. I guess the line breaks don't work inside jsx or something